pkg/netpol/connlist/connlist.go

```diff
@@ -49,10 +49,11 @@ const (
 	TxtFormat     = "txt"
 	JSONFormat    = "json"
 	DotFormat     = "dot"
+	CsvFormat     = "csv"
 )
 
 // ValidFormats array of possible values of output format
-var ValidFormats = []string{TxtFormat, JSONFormat, DotFormat}
+var ValidFormats = []string{TxtFormat, JSONFormat, DotFormat, CsvFormat}
```

Comment:
Should be `CSVFormat`, and `DOTFormat` same as `JSONFormat` - they're all acronyms. 
BTW - may want to use `TextFormat` explicitly